euphonic/qpoint_phonon_modes.py

```diff
@@ -265,10 +272,11 @@ def calculate_structure_factor(
         # Take mod squared and divide by frequency to get intensity
         sf = np.real(
             np.absolute(term*np.conj(term))/np.absolute(self._frequencies))
+        sf = sf/(2*self.crystal.n_atoms)
```

Comment:
I think that python has divide and assign /= operator, so `sf /= (2*....` could be neater.